src/main/scala/dpla/ingestion3/harvesters/file/NaraFileHarvestMain.scala

```diff
@@ -0,0 +1,224 @@
+package dpla.ingestion3.harvesters.file
+
+import java.io.{File, FileInputStream}
+import java.util.zip.GZIPInputStream
+
+import dpla.ingestion3.utils.Utils
+import org.apache.avro.Schema
+import org.apache.avro.file.{CodecFactory, DataFileWriter}
+import org.apache.avro.generic.{GenericData, GenericDatumWriter, GenericRecord}
+import org.apache.commons.io.IOUtils
+import org.apache.log4j.LogManager
+import org.apache.tools.bzip2.CBZip2InputStream
+import org.apache.tools.tar.TarInputStream
+import org.rogach.scallop.{ScallopConf, ScallopOption}
+
+import scala.util.{Failure, Success, Try}
+import scala.xml.{MinimizeMode, Node, Utility, XML}
+
+
+object NaraFileHarvestMain {
+
+  val schemaStr: String =
```

Comment:
@mdellabitta after PR #46 is merged in this should pull from those files, correct?